starfish/experiment/experiment.py

```diff
@@ -89,11 +118,70 @@ def name(self) -> str:
     def image_types(self) -> Set[str]:
         return set(self._images.keys())
 
-    def get_image(self, item: str, crop_parameters: Optional[CropParameters]=None) -> ImageStack:
-        return ImageStack.from_tileset(self._images[item], crop_parameters)
+    def show_aligned_image_groups(self):
```

Comment:
Would not call this `show_*` as it doesn't actually show anything.

also:
```suggestion
    def show_aligned_image_groups(self) -> dict:
```